src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleAutoBow.kt

```diff
@@ -224,6 +227,8 @@ object ModuleAutoBow : Module("AutoBow", Category.COMBAT) {
             tree(targetTracker)
             tree(rotationConfigurable)
         }
+        private val targetRenderer = tree(OverlayTargetRenderer(this.module!!))
```

Comment:
`ModuleAutoBow` would be better. `this.module!!` might cause bugs that are hard to investigate

Reply:
done.